tests/std/test.lst

```diff
@@ -255,6 +255,7 @@ tests\P0896R4_ranges_alg_mismatch
 tests\P0896R4_ranges_alg_move
 tests\P0896R4_ranges_alg_none_of
 tests\P0896R4_ranges_alg_search
+tests\P0896R4_ranges_alg_swap_ranges

```

Comment:
Git is complaining that you have a merge conflict in this file (a line that this file wants to change has been changed or is adjacent to a line changed by some other commit). You'll need to merge `master` into this branch and resolve the resulting conflict. In this case, the issue is that the `search_n` PR and this PR both want to add a line in the same place; we want to resolve it by adding both lines.